### PR TITLE
27.0.2

### PR DESCRIPTION
**Pull request by skjnldsv**

### CODE REVIEW

**Review by nickvergessen — APPROVED** [review]

**Comment by nickvergessen** [comment]
Failing tests are expected due to not available URL

**Comment by skjnldsv** [comment]
> Failing tests are expected due to not available URL

I'm setting back the `prereleases/nextcloud-27.0.2.zip` to ensure tests passes

File: config/config.php [hunk 1]
```diff
@@ -667,34 +667,34 @@
...
 			],
 		],
 		'26.0.5' => [
 			'100' => [
-				'latest' => '27.0.2 RC1',
-				'internalVersion' => '27.0.2.0',
-				'downloadUrl' => 'https://download.nextcloud.com/server/prereleases/nextcloud-27.0.2rc1.zip',
+				'latest' => '27.0.2',
+				'internalVersion' => '27.0.2.1',
+				'downloadUrl' => 'https://download.nextcloud.com/server/prereleases/nextcloud-27.0.2.zip',
```
Comment (AndyScherzinger) [inline config/config.php:688]: prereleases or releases folder?

File: config/config.php [hunk 2]
```diff
@@ -667,34 +667,34 @@
 	'beta' => [
 		'27' => [
 			'100' => [
-				'latest' => '27.0.2 RC1',
-				'internalVersion' => '27.0.2.0',
-				'downloadUrl' => 'https://download.nextcloud.com/server/prereleases/nextcloud-27.0.2rc1.zip',
+				'latest' => '27.0.2',
+				'internalVersion' => '27.0.2.1',
+				'downloadUrl' => 'https://download.nextcloud.com/server/prereleases/nextcloud-27.0.2.zip',
```
Comment (AndyScherzinger) [inline config/config.php:672]: prereleases or releases folder?

File: config/config.php [hunk 3]
```diff
@@ -110,34 +110,34 @@
...
 			],
 		],
 		'26.0.4' => [
 			'100' => [
-				'latest' => '27.0.1',
-				'internalVersion' => '27.0.1.2',
-				'downloadUrl' => 'https://download.nextcloud.com/server/releases/nextcloud-27.0.1.zip',
+				'latest' => '27.0.2',
+				'internalVersion' => '27.0.2.1',
+				'downloadUrl' => 'https://download.nextcloud.com/server/prereleases/nextcloud-27.0.2.zip',
```
Comment (AndyScherzinger) [inline config/config.php:131]: prereleases or releases folder?

**Review by AndyScherzinger — APPROVED** [review]

File: config/config.php (hunk 1 above)
Comment (skjnldsv) [reply config/config.php:688]: For now, prereleases
Will adjust Thursday when we also release 25 and 26 finals

**Comment by skjnldsv** [comment]
Tests are good, merging

**Pull request merged**